PROCESS.md

```diff
@@ -0,0 +1,109 @@
+   project in GitHub by a committer for tracking the status of all accepted 
+   proposals in one place.
+4. During development, work in progress may optionally be shared for feedback 
+   and early review via a 
+   [draft pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests).
+5. Upon completion of feature development, a pull request is created for review 
+   and 
+   [linked to the associated issue]((https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) for review.
+   Linking the issue and PR will result in the issue being closed automatically 
+   upon the PR being merged, and facilitate tracking of the proposed feature in
+   [Feature Development](https://github.com/AcademySoftwareFoundation/OpenColorIO/projects/3). 
+   See [CONTRIBUTING.md](CONTRIBUTING.md) for an overview of commit signing 
+   requirements and a guide on submitting a pull request.
+6. The pull request is merged after being approved as documented in the 
+   [Required Approvals](#Required-Approvals) section below.
+
+**NOTE**: Software design is often an iterative process with development. While 
+a design proposal is required, it is understood that development of the 
+proposed feature may begin prior to opening a design proposal issue in GitHub. It 
+is however highly encouraged to propose the design early in the interest of 
+respecting the contributor's time and effort. This avoids getting too far into 
+development without community involvement, which may lead to redesign and 
+extensive rework before changes can be approved.
+
+## Required Approvals
+
+Modifications of the contents of the OpenColorIO repository are made on a
+collaborative basis. Anyone with a GitHub account may propose a modification via
+pull request and it will be considered by the project Committers.
+
+Pull requests must meet a minimum number of Committer approvals prior to being
+merged. Rather than having a hard rule for all PRs, the requirement is based on
+the complexity and risk of the proposed changes, factoring in the length of
+time the PR has been open to discussion. The following guidelines outline the
+project's established approval rules for merging:
+
+* All changes must receive a minimum of two approvals from Committers 
+  listed in [COMMITTERS.md](COMMITTERS.md) that are not affiliated with the 
+  author or their company prior to being merged. GitHub branch protections 
+  are enabled to enforce minimum committer count and Committer status. A 
+  Committer approval is denoted in a PR with a green check mark. A gray check 
+  mark beside an approval means the approver does not count toward the minimum
+  approval count, but such approvals are welcome and encouraged from the 
+  community.
+
+* Ideally one of two approvers of a pull request should be the "owner" for that 
```

Comment:
The owner concept is definitively not clear. If needed, it must be based on active contributions (through pull request, etc.) and / or active involvement (through code review, issue review, etc.) in the designated code base.

Reply:
Good point. If we ever wanted to do this it can be achieved with a `CODEOWNERS.md` file which I believe GH supports for directing approvals, but agree that we're not doing this now so it should be removed.